### PR TITLE
add thrift service sdk

### PR DESCRIPTION
**Pull request by DongLiang-0**

### CODE REVIEW

**Comment by hf200012** [comment]
add license

File: thrift-service/pom.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        <artifactId>doris-sdk</artifactId>
+        <groupId>org.apache.doris</groupId>
+        <version>1.0.0</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>thrift-service</artifactId>
+
+    <properties>
+        <libthrift.version>0.13.0</libthrift.version>
```
Comment (zy-kkk) [inline thrift-service/pom.xml]: The most recent Doris thrift version is 0.16, and the 1.2 version is 0.13, so maybe we should use multiple versions

**Review by CalvinKirs — COMMENTED** [review]
i think you forgot this https://github.com/apache/doris-shade/blob/master/LICENSE

**Comment by CalvinKirs** [comment]
> i think you forgot this https://github.com/apache/doris-shade/blob/master/LICENSE↳

Sorry, I thought it was a first commit.

File: thrift-service/pom.xml (hunk 1 above)
Comment (DongLiang-0) [reply thrift-service/pom.xml]: done

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**